engine/runtime/src/main/java/org/enso/EnsoBoot.java

```diff
@@ -0,0 +1,50 @@
+package org.enso;
+
+import java.io.File;
+import java.net.URL;
+import java.net.URLClassLoader;
+
+public final class EnsoBoot {
+  private EnsoBoot() {}
+
+  public static void main(String[] args) throws Exception {
```

Comment:
With the changes introduced in 2cd617095e I can use following command to start Enso on GraalVM for JDK21:
```
enso.$ /graalvm-21/bin/java -p \
  $HOME/.cache/coursier/v1/https/repo1.maven.org/maven2/org/graalvm/polyglot/polyglot/23.1.0/polyglot-23.1.0.jar:$HOME/.cache/coursier/v1/https/repo1.maven.org/maven2/org/graalvm/truffle/truffle-api/23.1.0/truffle-api-23.1.0.jar:runtime.jar \
  -m org.enso.runtime/org.enso.EnsoBoot --run t.enso
```

right now it fails on missing `js.foreign-object-prototype`, but that should be smoothly solvable by adding Graal.js on the modulepath.